resources/migrations/000_migrations.yaml

```diff
@@ -1526,7 +1562,7 @@ databaseChangeLog:
                     nullable: false
               - column:
                   name: object
-                  type: varchar
+                  type: ${text.type}
```

Comment:
MySQL doesn't support `varchar`, only `varchar(n)`; not sure why this used to work, but it doesn't work anymore. So I replaced the places where we were doing this with `text` (`varchar` without `(n)` is basically the same as `text` on Postgres anyway)